site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+
+We acknowledge that Google vends a number of commonly used third-party scripts, including Google Tag Manager, YouTube embeds, and ReCaptcha to name a few. We acknowledge that a number of our scripts could have a lighter performance impact on the Core Web Vitals, and we are committed to exploring ways to improve this impact where possible.
+
+# How can Chrome help?
+
+Third-party resources being poor for performance is often a tragedy of the commons requiring a step-change in the underlying ecosystem dynamics. Chrome wants to drive innovation to achieve the following outcomes:
+
+1.  ***Find better ways of loading third-party resources on the web while not regressing user-experience or business outcomes.***
+
+    We know that we cannot proceed far in this effort if we don't collaborate with partners, businesses, third-parties, and developers. We want to create an open field to discuss the possibilities and exchange ideas via public explainers and specifications. Developers will have time to provide feedback and test the impact of many of these proposals.
+
+2.  ***Enable users of third-party scripts to have better attribution for their costs in tooling and in the field, clear, well-paved paths for their usage, and better incentives during authoring time to ensure they are optimal by default.***
+
+    We want to enhance all layers, such as user agents, frameworks, and third-party scripts to reduce the performance impact of third-parties. We intend to also provide sufficient insights to help site owners employ best practices around each script embedded, including faster alternatives where applicable.
+
+## Our Approach
+
+We propose a three-pronged approach for achieving these outcomes..
+
+1.  **Give ***developers*** deeper attribution into per third-party impact via RUM and in Google's developer tooling.**
+
+    RUM refers to real user metrics data (also known as [field data](https://web.dev/lab-and-field-data-differences/#field-data)) available through [CrUX](https://developers.google.com/web/tools/chrome-user-experience-report/api/reference) and [web performance monitoring APIs](https://web.dev/vitals-measurement-getting-started/#collecting-rum-data). Google's developer tooling includes Lighthouse, and Chrome DevTools. We propose to enhance these APIs and tools so that site developers understand the impact of every third-party they have used on every page. The tools will also educate them about actions they can take to mitigate the impact (for example, deferring, facades, etc) and explore other potential solutions (other third-parties or DIY) with trade-offs. For the web performance monitoring APIs, we're exploring ways in which we can expand their coverage of cross-origin resources without compromising our users' privacy and security.
```

Comment:
```suggestion
    RUM refers to real user metrics data (also known as [field data](https://web.dev/lab-and-field-data-differences/#field-data)) available through [CrUX](https://developer.chrome.com/docs/crux/api/) and [web performance monitoring APIs](https://web.dev/vitals-measurement-getting-started/#collecting-rum-data). Google's developer tooling includes Lighthouse, and Chrome DevTools. We propose to enhance these APIs and tools so that site developers understand the impact of every third-party they have used on every page. The tools will also educate them about actions they can take to mitigate the impact (for example, deferring, facades, etc) and explore other potential solutions (other third-parties or DIY) with trade-offs. For the web performance monitoring APIs, we're exploring ways in which we can expand their coverage of cross-origin resources without compromising our users' privacy and security.
```

Reply:
The CrUX API link has been redirected to a doc on d.c.c., so I figure we can just skip the redirect and go straight to the final hop.

Reply:
SGTM!